src/ucp/core/ucp_ep.c

```diff
@@ -2668,6 +2665,28 @@ void ucp_ep_get_tl_bitmap(ucp_ep_h ep, ucp_tl_bitmap_t *tl_bitmap)
     }
 }
 
+void ucp_ep_get_lane_info_str(ucp_ep_h ucp_ep, ucp_lane_index_t lane,
+                              ucs_string_buffer_t *lane_info_strb)
+{
+    ucp_rsc_index_t rsc_index;
+    uct_tl_resource_desc_t *tl_rsc;
+
+    if (lane != UCP_NULL_LANE) {
```

Comment:
1. why NULL_LANE is printed as "wireup lane"
2. can we arrange this function as "if (lane=..) else if (lane=...) else .."
3. any common code with ucp_ep_print_info ?

Reply:
> 1. why NULL_LANE is printed as "wireup lane"

re-name to "NULL lane"

> 2\. can we arrange this function as "if (lane=..) else if (lane=...) else .."

done

> 3\. any common code with ucp_ep_print_info ?

unfortunately, no. `ucp_ep_print_info()` prints more info